lib/index.js

```diff
@@ -13,28 +36,7 @@ class Timeout {
 
     this._timeout = null;
     this._ref = true;
-    this.start(args);
-  }
-  start (args) {
-    const max = module.exports._TIMEOUT_MAX; // Use the exported value for testing purposes.
-
-    if (this._delay <= max) {
-      this._timeout = setTimeout(this._callback, this._delay, ...args);
-    } else {
-      const callback = () => {
-        this._delay -= max;
-        this.start(args);
-      };
-
-      this._timeout = setTimeout(callback, max, ...args);
-    }
-
-    if (this._ref === false) {
-      this._timeout.unref();
-    }
-  }
-  close () {
-    clearTimeout(this._timeout);
+    start.call(this, args);
```

Comment:
At this point, why even bother with `call`? Just pass `this` in as the first argument, and then `args`.